src/components/Deliberation/InterviewerView.js

```diff
@@ -278,7 +322,12 @@ class InterviewerView extends Component {
             const accepted = positiveVotes / allVotes.length >= 0.75;
             if (accepted) {
               const ref = this.props.firebase.application(id);
-              batch.update(ref, { "deliberation.accepted": true });
+              batch.update(ref, {
```

Comment:
Will we always need to set this data about provisional stuff here? This seems to be the function for just sending results in general, perhaps there should be a conditional gating which data is added to the update? Or is there a reason to always add the data here?

Reply:
Fixed in 32ec323b07c251c4abd46a6c3c39198d223fd440